src/services/withdrawal.py

```diff
@@ -31,10 +31,10 @@ def get_finalization_batches(
         withdrawal_vault_balance: Wei,
         el_rewards_vault_balance: Wei
     ) -> list[int]:
-        is_paused = self._fetch_is_paused()
-        has_unfinalized_requests = self._has_unfinalized_requests()
+        if self._fetch_is_paused():
```

Comment:
Let's add the comment that we are not stopping txs here, we just don't eject

Reply:
I remember that we already discussed, but I don't remember this results of discussion

Reply:
Or we decide to stop tx?

Reply:
It's withdrawal finalization not ejecting. In this case we r not stopping tx just finalizing zero requests